src/main/modules/document-download/index.ts

```diff
@@ -9,26 +9,47 @@ const proxy = require('express-http-proxy');
 
 export class DocumentDownloadMiddleware {
   public enableFor(app: Application): void {
-    const addToReqPath = (req: AppRequest) => {
+    const addApplicationToReqPath = (req: AppRequest) => {
       return req.session.userCase?.documentsGenerated?.find(doc => doc.value.documentType === DocumentType.APPLICATION)
         ?.value.documentLink.document_binary_url;
     };
 
+    const addRespondentAnswersToReqPath = (req: AppRequest) => {
+      return req.session.userCase?.documentsGenerated?.find(
```

Comment:
do you need the ? on req.session.userCase?

Reply:
nope, I'll remove it